packages/core/src/js/client.ts

```diff
@@ -83,19 +82,6 @@ export class ReactNativeClient extends BaseClient<ReactNativeClientOptions> {
     });
   }
 
-  /**
-   * Sends user feedback to Sentry.
-   */
-  public captureUserFeedback(feedback: UserFeedback): void {
```

Comment:
I would only deprecate this, to avoid breaking changes. Since the Client class is public interface.

https://github.com/getsentry/sentry-react-native/blob/8ab11b68f5637b8a7886341bcc06214f8cd48ebb/packages/core/src/js/index.ts#L60